tests/api.c

```diff
@@ -2362,6 +2366,23 @@ static void test_wolfSSL_PKCS12(void)
 #endif /* OPENSSL_EXTRA */
 }
 
+/* Testing functions dealing with PKCS5 */
+static void test_wolfSSL_PKCS5(void)
+{
+#if defined(OPENSSL_EXTRA) && !defined(NO_SHA) && !defined(NO_PWDBASED)
+    const char *passwd = "pass1234";
+    const unsigned char *salt = (unsigned char *)"salt1234";
+    unsigned char *out = (unsigned char *)XMALLOC(SHA_DIGEST_SIZE, NULL,
```

Comment:
Please add a check for the XMALLOC on out here. Thanks

Reply:
I added AssertNotNull check.